### DISCUSSION TITLE
Store last accessed tab in preferences

### DISCUSSION THREAD

**Discussion by jmikrut** [payloadcms/payload#1239] [Feature Requests & Ideas]
We should store the last accessed tab in user preferences, to automatically restore the tab back to the last used tab when a visitor returns to a document.

Marking as planned.

**Comment by mfilteau** [comment]
Hi James,
Could we have an option to disable this behavior too? I have a case where the first tab is the primary information the user is looking for. Now, depending on the history of access to the document, the tab shown may not be the first. This is confusing for many users.
